security/security-spring/src/main/java/org/cbioportal/security/spring/CancerStudyPermissionEvaluator.java

```diff
@@ -181,6 +182,15 @@ public boolean hasPermission(Authentication authentication, Serializable targetI
                 }
             }
             return true;
+        } else if ("List<SampleListId>".equals(targetType)) {
```

Comment:
@ersinciftci and I agreed it should be that way for CancerStudy lists, because when they are passed as a parameter they should have the correct cancer study list already (they should first get the filtered list, then use that filtered list to get other information).  In general that *should* be true with other lists as well, but we definitely should think carefully about each case to make sure it is true.